### DISCUSSION TITLE
Looking for good first issue

### DISCUSSION THREAD

**Discussion by GlennWSo** [jimy-byerley/pymadcad#55] [General]
Hi,

Think this project is awsome, so i wanna help. Im looking for good first issue to get myself a bit more familiar with the plumbing.

thx :)

**Comment by jimy-byerley** [comment]
Great news ! And thanks for the feedback !
What would you prefer: a feature to implement or a bug to solve ?

**Reply by GlennWSo** [reply]
Maybe a bug would be best, since my goal right now is to familiarize myself with existing stuff

**Reply by jimy-byerley** [reply]
Then what would you think of #56  :) ?
That's part of the already known issues we noticed on madcad, but that I never took time to write on the repo

**Reply by GlennWSo** [reply]
#56 its perfect! I will try to resolve this :)
